ts/Core/Renderer/HTML/AST.ts

```diff
@@ -243,6 +243,33 @@ class AST {
         }
     }
 
+
+    public static serialize(node: Highcharts.ASTNode): string {
```

Comment:
@TorsteinHonsi Any objections to moving this `serialize` functionality to the AST class?